apps/voting/app/src/App.js

```diff
@@ -207,7 +192,7 @@ class App extends React.Component {
           </AppLayout.Header>
           <AppLayout.ScrollWrapper>
             <AppLayout.Content>
-              {displayVotes ? (
+              {appStateReady && votes.length > 0 ? (
```

Comment:
I was thinking it would also be nice to disable the top bar and empty state card's action if the app's not ready yet (requires https://github.com/aragon/aragon-ui/pull/219).